sample_apps/simple_app/example_app.cc

```diff
@@ -104,16 +104,6 @@ void server_application(secure_authenticated_channel& channel) {
   channel.write(strlen(msg), (byte*)msg);
 }
 
-bool run_me_as_server(const string& host_name, int port,
```

Comment:
Discovered that this is an unused function. So, deleted it. (Let me know if you see any further need for this such that we should retain it.)